.github/workflows/ci.yaml

```diff
@@ -0,0 +1,71 @@
+# Copyright 2022 Canonical Ltd.
+# See LICENSE file for licensing details.
+name: Tests
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+on:
+  pull_request:
+  schedule:
+    - cron: '53 0 * * *' # Daily at 00:53 UTC
+  # Triggered on push to branch "main" by .github/workflows/release.yaml
+  workflow_call:
+
+jobs:
```

Comment:
Yes, this was on purpose -- as the charm was using a version of `data_interfaces` that wasn't yet merged.

I think I thought 2 do it in a next PR but since that's now available, I could put back the check.

Reply:
@juditnovak then just file a bug in this repo and link it here, so we can keep track. Otherwise, we can resolve this conversation.

Reply:
No, that's good, I rather put it back. IT's a simple thing and then I have it off the way. Should B good now.

Reply:
@juditnovak reopening as I do not see it yet. Can you add the lib-check here or open a bug so we can track this task for later?

Reply:
Bah.

Addressing it in a separate commit given that libs are broken. And I'd REALLY like to have a green pipeline for the core charm functionality PR.

I made an attempt: https://github.com/canonical/opensearch-dashboards-operator/actions/runs/8628051172

Hitting bug discussed [here](https://chat.canonical.com/canonical/pl/qj8gu7m16t8ztnc3gqjde3pwme) and [here](https://chat.canonical.com/canonical/pl/rkpqd418o78q7k44bd76zpfrjo).

The underlying reason should be a Juju bug: https://bugs.launchpad.net/juju/+bug/2058012

Reply:
(Even worse -- I realize that latest `data_interfaces` introduce issues as well. All taken on [DPE-4041](https://warthogs.atlassian.net/browse/DPE-4041).)